seedServer/src/main/java/facades/UserFacade.java

```diff
@@ -47,6 +48,22 @@ public IUser getUserByUserId(String id) {
           em.close();
       }
   }
+  
+  public void setUser(String username, String password) throws PasswordStorage.CannotPerformOperationException{
+      EntityManager em = getEntityManager();
+      try {
+          em.getTransaction().begin();
+          Role userRole = new Role("User");          
+          User user = new User(username, password );
+          user.addRole(userRole);   
+          em.persist(user);         
+          em.getTransaction().commit();
+          System.out.println("Created New user");
```

Comment:
Delete the print line, it's good for quick debug, but not for committed code.

Reply:
Done